src/main/java/com/thebois/views/ViewportWrapper.java

```diff
@@ -0,0 +1,27 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+/**
+ * Wrapper class for viewports.
```

Comment:
Since the class only wraps a *single* viewport it's probably better to not pluralize it here.

Reply:
Makes sense.